code/modules/projectiles/guns/projectile/automatic/c20r.dm

```diff
@@ -13,21 +16,21 @@
 	mag_well = MAG_WELL_SMG
 	magazine_type = /obj/item/ammo_magazine/smg
 	auto_eject = TRUE
-	matter = list(MATERIAL_PLASTEEL = 20, MATERIAL_PLASTIC = 12)
-	price_tag = 3000
+	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_STEEL = 4, MATERIAL_PLASTIC = 6)
+	price_tag = 1800
 	auto_eject_sound = 'sound/weapons/smg_empty_alarm.ogg'
 	unload_sound 	= 'sound/weapons/guns/interact/sfrifle_magout.ogg'
 	reload_sound 	= 'sound/weapons/guns/interact/sfrifle_magin.ogg'
 	cocked_sound 	= 'sound/weapons/guns/interact/sfrifle_cock.ogg'
-	zoom_factor = 0.2
-	damage_multiplier = 1.1
```

Comment:
Why ``damage_multiplier`` was changed to ``penetration_multiplier``? Even is it's 1.0, leave it just like that for code readability.

Reply:
Adressed.